### ISSUE TITLE
Upgrade rest package to use latest aries deps

### ISSUE THREAD

**Issue by dblane-digicatapult** [hyperledger/aries-framework-javascript-ext#215]
We are interested in upgrading the rest package to use the latest available versions of `aries-framework/core` and `aries-framework/node`. Our primary driver for this is because as far as we can tell earlier versions do not support the abstraction of hyperledger indy.  We believe that it is important to display a variety of anoncreds registries and would benefit other users of this REST API.

We have read your contributing guidelines and will conform to the standards set out within.

Can you please indicate if you would like us to proceed with this work or if you have other plans as to why we should not proceed?

**Comment by TimoGlastra** [comment]
Hey @dblane-digicatapult! Thanks for opening this issue. It's been on our todo list for a long time to update the rest wrapper, but just haven't got around to it.

I just finished wrapping up the updates for push-notifications, react-hooks, redux-store, so if we can also get the rest package to 0.4.0 that would be awesome.

So, definitely would like you to proceed with the work. 👍

**Comment by mattdean-digicatapult** [comment]
Hi @TimoGlastra. I'm currently working with @dblane-digicatapult on this and had a couple of questions. Specifically whether this should use `indy-sdk` and continue to use the legacy schema/credential-definition formats from that module or should we try to swap out for `indy-vdr`, `anoncreds`, `askar` etc.?

It seems to me that the latter is clearly the path forward, but this would cause breaking changes to the API presented by the `rest` package. I'm also struggling to find docs on even using `indy-sdk` after the restructure which is making me a bit wary of that approach.

Basically are you happy for me to go straight to `indy-vdr` and the generic `anoncreds` modules even though it will lead to some breaking changes?

**Comment by mattdean-digicatapult** [comment]
Fabulous @TimoGlastra, working towards a PR currently. Hopefully have something for you to look at in the next few days. Thanks for the guidance

**Comment by djleonskennedy** [comment]
@dblane-digicatapult @TimoGlastra will be update 0.4.0 also based on tsoa?
